wallet/src/de/schildbach/wallet/service/CoinJoinService.kt

```
@@ -196,7 +196,11 @@ class CoinJoinMixingService @Inject constructor(
                 // Time has changed, handle the change here
                 log.info("Time or Time Zone changed")
                 coroutineScope.launch {
-                    updateTimeSkewInternal(getTimeSkew(force = true))
+                    try {
+                        updateTimeSkewInternal(getTimeSkew(force = true))
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)